bootstrap/src/index.ts

```diff
@@ -77,7 +78,10 @@ const executeSync = (execute: ExecuteWrappedResponse): ExecuteSync => {
     return withMiddleware(execute)
       .then((executeWithMiddleware) => executeWithMiddleware(data))
       .then((result) => callback(result.statusCode, result.data))
-      .catch((error) => callback(error.statusCode || 500, Requester.errored(data.id, error)))
+      .catch((error) => {
+        logger.error(error.toString(), { stack: error.stack })
```

Comment:
Please add this to the available `withLogger`, so we don't interrupt this specific flow. There you can `try {...} catch`  the error to log it and throw again.